concepticondata/concepticon.tsv

```diff
@@ -3760,3 +3760,6 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3774	ONE THOUSAND AND TWO	Quantity	The natural number one thousand and two (1002).	Number	
 3775	ONE THOUSAND AND SIXTY	Quantity	The natural number one thousand and sixty (1060).	Number	
 3776	TWO THOUSAND AND TWO	Quantity	The natural number two thousand and two (2002).	Number	
+3777	CATFISH	Animals	Any of ray-finned fish with long stiff barbels, resembling a cat's whiskers. There are diverse types of catfish, ranging in size and appearance and inhabiting mostly freshwater environments.	Person/Thing	
+3778	DAUGHTER (OF MALE EGO)	Kinship	A daughter, as referred to by her father.	Person/Thing	
+3779	SON (OF MALE EGO)	Kinship	A son, as referred to by his father.	Person/Thing	
```

Comment:
So now I'm officially confused. The original motivation behind the naming was the following: the 'ego' left by the author in parenthesis means that this concept relates to one specific instance of referring to a person as your daughter/son if you are yourself a man (i.e. father). Following this argumentation, another UNRELATED MALE person can't call the person referred to as daughter or son with these lexemes. This is what I thought refers to by 'ego'. Or I'm just overthinking :)

Reply:
I thought it was similar to BROTHER (F. EGO) 560 BROTHER (OF WOMAN) but you could also be right.

Reply:
Alright, once I've looked at the definition of BROTHER (OF WOMAN), I've understood what you mean. The reason why the naming involving (OF WOMAN) can be potentially ambiguous (at least for me, maybe it's not the case for others), is that it can mean:

1. referring to the person in question if you are a woman. In this case, without looking at the definition of BROTHER (OF WOMAN), I would assume this concept can be used by potentially any woman when referring to somebody's brother.

2. referring to the person in question from the ego (deictic) perspective. In this case BROTHER (OF WOMAN) would mean that this concept can be used exclusively by the sister of the person in question.

I hope I didn't confuse things even more :)

Reply:
Kinship terms are confusing by nature ;) Most systems are far more intricate than English and therefore, we lack a good term to describe the exact relationship. But I'd propose you follow @LinguList suggestion to use the "ego" style for now.